cli/src/swam/cli/Server.scala

```diff
@@ -0,0 +1,171 @@
+package swam
+package cli
+
+import scala.collection.mutable.ListBuffer
+
+// Simple server
+import java.io._
+import java.net.{ServerSocket, Socket}
+import java.nio.ByteBuffer
+import java.nio.file.Path
+
+import cats.effect.IO
+import swam.optin.coverage.{CoverageListener, CoverageReporter}
+import swam.runtime.{Function, Value}
+
+import scala.util.control.Breaks._
+
+// TODO: Read port from env-variable SWAM_SOCKET_PORT
+object Server {
+  val maxQueue = 50000
```

Comment:
This looks really interesting. Just a thing: I'd like to keep Swam pure and functional. Can you try to implement the server using `fs2-io`? See https://fs2.io/io.html

If you are not used to this kind of programming I can do it for you, but I'd really love to have this before merging upstream.

Reply:
Yeah, I get that. I'll have a go at it..!